.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+  workflow_run:
+    workflows: [CI]
+    branches: [main]
+    types:
+      - completed
+
+jobs:
+  Semantic_Release:
```

Comment:
We should probably stick to a naming convention for jobs. I see we call the CI job `CI_PIPELINE` (all caps). So for consistency maybe do the same here?

Reply:
Renamed CI_PIPELINE to CI_Pipeline. All Caps didn't look right in the UI. And no other Actions workflows I've seen so far uses all caps